tests/post_training/pipelines/lm_weight_compression.py

```diff
@@ -193,20 +205,29 @@ def _dump_model_fp32(self) -> None:
         to the dedicated shared folder.
         """
         self.model_hf.save_pretrained(self.fp32_model_dir)
-        self.model_hf._save_config(self.fp32_model_dir)
+        if self.backend != BackendType.TORCH:
+            self.model_hf._save_config(self.fp32_model_dir)
 
     def _compress(self):
         """
         Actual call of weight compression
         """
+        if self.backend == BackendType.TORCH:
+            from nncf.torch.model_creation import is_wrapped_model
+            from nncf.torch.model_creation import wrap_model
+
+            if not is_wrapped_model(self.model):
+                example_input = next(iter(self.calibration_dataset.get_inference_data()))
+                self.model = wrap_model(self.model, example_input=example_input, trace_parameters=True)
+                self.calibration_dataset = None
+
         self.compressed_model = nncf.compress_weights(
             self.model,
             dataset=self.calibration_dataset,
             **self.compression_params,
         )
 
     def _validate(self):
-        is_stateful = self.params.get("is_stateful", False)
```

Comment:
It looks like `_validate` does not require any changes.

Reply:
Done